addons/sys_zeus/XEH_postInit.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+private _category = format ["ACRE %1", localize "str_a3_cfghints_curator_curator_displayname"];
+
+[_category, "ZeusTalkFromCamera",  [(LLSTRING(SpeakFromCamera)), (LLSTRING(SpeakFromCamera_description))],
+    { call FUNC(handleZeusSpeakPress) },
+    { call FUNC(handleZeusSpeakPressUp) },
+[40, [false, false, false]]] call cba_fnc_addKeybind; //Default bound to `
+
+// There is no EH for opening the zeus interface
```

Comment:
You can do it via CBA XEH for Displays, like here: https://github.com/acemod/ACE3/blob/e5a15d200f44df5fccc0bc5575d18d80b35538dd/addons/spectator/CfgEventHandlers.hpp#L27-L29
Which is effectively an EH for opening the Zeus interface.